.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+extend-ignore = E203, W503
```

Comment:
It would be great if you could add a comment explaining the need for these ignores (and when it would be save to remove them.

Reply:
Not the OP but the line length and E203 come from [black's recommended config for flake8](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#configuration). [W503 is "line break before binary operator"](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes) but apparently it is not universally accepted and is not included in pep8.

Reply:
yes, those came from the black recommendations. I guess W503 used to be recommended but I guess has been turned off by default in current flake8 releases. I can add a comment to the flake8 and isort configs, and use the newer `profile = black` setting for isort.